golang/lib/analytics_logger/analytics_logger.go

```diff
@@ -0,0 +1,24 @@
+package analytics_logger
```

Comment:
added this as this will be shared in CLI / APIC now, earlier this was in CLI only